docs/form-question-types.rst

```diff
@@ -140,6 +140,20 @@ The external app widget is displayed when the ``appearance`` attribute begins wi
 
   text,ex_string_widget,Ex string widget,ex:change.uw.android.BREATHCOUNT,text type with ex:change.uw.android.BREATHCOUNT appearance (can use other external apps)
 
+.. _thousands-sep:
```

Comment:
You may not need an anchor here but if you want to have one it should be related to the section!

Reply:
Right, fixed.